src/api/dataops/queries/baseQuery.ts

```diff
@@ -40,7 +35,7 @@ export abstract class BaseQuery<T> {
 
   protected constructor(
       protected queryExecuter: RequestExecuter,
-      private readonly action: QueryAction,
+      public readonly action: QueryAction,
```

Comment:
Why do you need action to be public?

Reply:
to use it in `InsertQuery` (and in tests). as it's a `readonly` property I see no problem doing it.

Reply:
what about `protected`? In tests we are using `as any` if we need to get access to a private member, I see no problem with that 
I don't want to use `public` because it shows that we going to use this member somewhere outside the class, which is not true in this case

Reply:
changed to `protected`. man, we need to get rid of all these inheritance :sweat_smile: